README.md

```diff
@@ -1,4 +1,5 @@
 ## [Shadowsocks](https://shadowsocks.org) for Android
+### Shadowsocks is a secure [socks5](https://en.wikipedia.org/wiki/SOCKS#SOCKS5) proxy, designed to protect your Internet traffic.  It is available on multiple platforms, but this repository is for the Android platform specifically.  Before installing Shadowsocks, please read our [privacy policy](https://github.com/shadowsocks/shadowsocks-android/blob/master/privacy_policy.md).
```

Comment:
Do not use subtitles `###` for a long sentence.

Reply:
Removed.